### PR TITLE
fix: add google chat keys to ping editorial team

### PR DESCRIPTION
**Pull request by scissorsneedfoodtoo**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

This PR adds the missing Google Chat secrets to the `deploy-eng` workflow so the Editorial team can get a ping if there's an issue with duplicate slugs in the future.

There are also some tests now just to verify that, whenever there are posts with duplicate slugs between the two CMSs, that we only build the older version of that post.

### CODE REVIEW

File: utils/ping-editorial-team.js [hunk 1]
```diff
@@ -21,12 +21,13 @@ WARNING: Duplicate Posts Found
 -----------------------------------------------
 ${msg}
 `);
+  console.error(msg);
```
Comment (ojeytonwilliams) [inline utils/ping-editorial-team.js]:
```suggestion
```
Hey, Kris, this isn't a proper review, but I was skimming over it and noticed this!

Comment (scissorsneedfoodtoo) [reply utils/ping-editorial-team.js]: Thanks for catching this! Totally forgot about this line.

**Review by raisedadead — APPROVED** [review]
